fingerprint.c

```diff
@@ -315,10 +321,10 @@ static int fingerprint_cancel(struct fingerprint_device *dev)
 
     ALOGI("%s : -",__func__);
 
-    /*fingerprint_msg_t msg;
```

Comment:
I'm not entirely convinced here - this was commented out because apparently you're not supposed to do it since like android 7.1 or something..

Reply:
I got an error message in logcat with error in cancelling authentication function, I thought this was causing it, when I uncomment this I got no error, I'll look further in this.

Reply:
Any news on this?